oncotator/output/RecordBuilder.py

```diff
@@ -289,6 +289,11 @@ def addInfo(self, sampleName, ID, num=None, dataType="String", val=None, isSplit
             self._determineVal2FixedNumField(self._info, ID, nalts, isSplit, val)
         elif num == 0:  # num is either true or false
             if ID not in self._info:
+
```

Comment:
Are you sure you mean ``val == ''``?  And not ``val is None``?  Also, were you able to make a unit test?  That should be really easy.